src/Game.cpp

```diff
@@ -815,9 +815,9 @@ void Game::LoadMap(const std::filesystem::path& path)
 void Game::LoadLandscape(const std::filesystem::path& path)
 {
 	ecs::systems::InitializeLevel();
-	if (_landIsland)
+	if (Locator::terrainSystem::has_value())
 	{
-		_landIsland.reset();
+		Locator::terrainSystem::reset();
```

Comment:
There should only be one reset in the whole lifetime of game

Reply:
fixed on the next commit